migracja.rst

```diff
@@ -0,0 +1,236 @@
+**Laboratorium EE**
+
+*30 XII 2016*
```

Comment:
Uważam, że wszystkie dokumenty powinny mieć daty intencjonalnie wpisane przez Autorów. Nawet jeśli potem są zmiany, data daje jakieś pojęcie, kiedy sprawą się zajmowano. Jeśli zmiany są istotne, można datę aktualizować. Daty Gita to nie to samo.